boards/feather_m0/Cargo.toml

```diff
@@ -28,6 +28,15 @@ optional = true
 version = "0.3"
 optional = true
 
+[dependencies.rtt-target]
+version = "0.3.1"
+features=["cortex-m"]
+optional = true
+
+[dependencies.systick-monotonic]
+version = "1.0.0"
+optional = true
```

Comment:
As this is a dependency only in an example, it should go in dev-dependencies

Reply:
Same with rtt stuff I think

Reply:
@TDHolmes thank you for your suggestions. I applied them to the PR.